components/Footer/index.js

```diff
@@ -0,0 +1,67 @@
+import Link from "next/link";
+import React from "react";
+import styled from "styled-components";
+
+export default function Footer() {
+  return (
+    <StyledFooter>
+      <div class="wrapper">
+        <div className="top_nav"></div>
+      </div>
+      <div class="bottom_nav">
```

Comment:
`class` shouldn't be used as a prop in React. `className` would be the equivalent, but since we're using styled components, you'd typically style them without the use of `className`.

Reply:
You are totally right. forgot to remove all residue from all the previous changes. All set now!